cmd/subnetcmd/upgradecmd/apply.go

```diff
@@ -97,34 +116,11 @@ func applyCmd(_ *cobra.Command, args []string) error {
 
 // For a already deployed subnet, the supported scheme is to
 // save a snapshot, and to load the snapshot with the upgrade
-func applyLocalNetworkUpgrade(subnetName string, sc models.Sidecar) error {
-	// if there's no entry in the Sidecar, we assume there hasn't been a deploy yet
-	networkKey := models.Local.String()
-	if sc.Networks[networkKey] == (models.NetworkData{}) {
-		return subnetNotYetDeployed()
-	}
-	// let's check update bytes actually exist
-	netUpgradeBytes, err := app.ReadUpgradeFile(subnetName)
-	if err != nil {
-		if err == os.ErrNotExist {
-			ux.Logger.PrintToUser("No file with upgrade specs for the given subnet has been found")
-			ux.Logger.PrintToUser("You may need to first create it with the `avalanche subnet upgrade generate` command or import it")
-			ux.Logger.PrintToUser("Aborting this command. No changes applied")
-		}
-		return err
-	}
-
-	// read the lock file right away
-	lockUpgradeBytes, err := app.ReadLockUpgradeFile(subnetName)
-	if err != nil {
-		// if the file doesn't exist, that's ok
-		if !os.IsNotExist(err) {
-			return err
-		}
+func applyLocalNetworkUpgrade(subnetName, networkKey string, sc *models.Sidecar) error {
```

Comment:
do we need networkKey as param? always the same

Reply:
you are right, we don't need it. I thought though it to be nice that both funcs have the same signature